wcivf/apps/elections/devs_dc_client.py

```diff
@@ -9,7 +9,7 @@ class DevsDCAPIException(Exception):
     def __init__(self, response: requests.Response):
         try:
             self.message = {"error": response.json().get("message", "")}
-        except JSONDecodeError:
+        except (JSONDecodeError, AttributeError):
             self.message = ""
         self.status = response.status_code
         self.response = response
```

Comment:
Do these lines mean that a 404/`Not found` will be returned to the user?

Reply:
I think _any_ error is simply caught and a generic error is given on the home page